### DISCUSSION TITLE
Readme的三张预览图很漂亮啊，想问下用什么软件可以做出这样的效果？

### DISCUSSION THREAD

**Discussion by tuutoo** [vbenjs/vue-vben-admin#1731] [Q&A]
请教一下大佬是用什么软件做的这个效果，挺赞的。

![image](https://user-images.githubusercontent.com/9068314/159223133-913dc097-8045-4241-b400-cf41b4fbe6c3.png)

谢谢

**Comment by v012345** [comment]
ps
